catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/StdidcAttribute.java

```diff
@@ -103,6 +101,22 @@
 
   private static final Map<String, String> FIPS_TO_ISO3_MAP = getFipsMap();
 
+  /*
+   * Normalized attributes. These taxonomy terms will be duplicated by `ext.nitf.stdidc.*` when appropriate.
+   */
+
+  static final StdidcAttribute COUNTRY_ALPHA3_ATTRIBUTE =
+      new StdidcAttribute(
+          Location.COUNTRY_CODE,
+          COUNTRY_SHORT_NAME,
+          tre -> getAlpha3CountryCode(TreUtility.convertToString(tre, COUNTRY_SHORT_NAME)),
+          new LocationAttributes().getAttributeDescriptor(Location.COUNTRY_CODE),
+          "");
```

Comment:
This attribute was updated to use the `Normalized attribute` syntax that is used in other `xxxAttribute` classes, like https://github.com/codice/alliance/blob/1faf1f4dd3bb7262cc4088dc74b6b63b7b059519/catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/NitfHeaderAttribute.java#L165-L171.

One of the benefits, besides consistency, is to use the already-existing `location.country-code` `AttributeDescriptor` instead of defining it in this class. The corresponding constructor was also added.